lib/modules/touch.js

```diff
@@ -0,0 +1,50 @@
+import { contentLoaded } from '../utils';
+
+export default (shower) => {
+    contentLoaded(() => {
+        let action = false;
+        let clickable = false;
+
+        document.addEventListener('touchstart', (ev) => {
+            if (ev.touches.length > 1) {
+                action = false;
+                return true;
+            }
+            const touch = ev.touches[0];
+            const x = touch.clientX;
+            const { target } = touch;
+            clickable =
+                target.getAttribute('onclick') != null || target.getAttribute('href') != null;
```

Comment:
This will work correctly only for inline event handlers, not the `addEventListener`. There is no 100% reliable way to detect if an element is clickable. In Shower, we use `element.tabIndex !== -1`, which will nicely handle links / buttons / other built-in elements, as well as sane / a11y-friendly user-land controls.